analysis/analysis.py

```diff
@@ -34,6 +34,96 @@ def load_data_csv(csv_path: Path):
     return out_data
 
 
+def fetch_responses(dataset):
+    """
+    Extracts text responses to "In three words or fewer, what is this text about?" from given csv
+    file, depending on context given (story vs. ad). Returns lists of words used in question
+    responses.
+    :param dataset: list of loaded data to analyze
+    :return: response_ad and response_story, lists of strings of all words used in user responses
+    (duplicates included)
+    """
+    # cycling through dataset to find prompt responses, sorting based on context
+    responses_ad = []
+    responses_story = []
+    question = "In three words or fewer, what is this text about?"
+    for resp in dataset:
+        # removing punctuation and making string lowercase to standardize
+        resp['response'] = resp['response'].replace('.', '').lower()
```

Comment:
I'd move this data cleaning out into its own function, so it can be used elsewhere in the system, and so that it can be extended.

Reply:
Moved to function `clean_resp_strings`, returns a dataset with response text all lowercase and without commas and periods